static/bidder-info/aax.yaml

```diff
@@ -0,0 +1,19 @@
+maintainer:
+  email: product@aax.media
+gvlVendorID: 720
+modifyingVastXmlAllowed: true
+capabilities:
+  app:
+    mediaTypes:
+      - banner
```

Comment:
I see that in the [docs PR](https://github.com/prebid/prebid.github.io/pull/3696/files#diff-058aaef16b619c8a9e1279c95c108e70e2ffbd23237763a2f82042b82bd2feb0R7) you have mentioned support only for `banner` and `native` but here you've `video` as well. Can you please confirm if it's the docs PR that needs to be updated or does this PR needs an update to remove `video` from this list?

Reply:
@mansinahar We have added video media type in doc as well. Thanks